src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
Since the `RegistrationServiceImpl` instance is not expensive to create and does not maintain any state between tests, consider using `@BeforeEach` instead of `@BeforeAll` to ensure isolation between tests.